### PR TITLE
NXP-28554: fix saved sarches saving

### PR DESCRIPTION
**Pull request by richardsd**
My proposal is to add again the parse in the fetch method.

### CODE REVIEW

**Comment by richardsd** [comment]
Pipeline at https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/573/
and functional test at https://github.com/nuxeo/plugin-nuxeo-web-ui/pull/141

**Comment by richardsd** [comment]
I've detected a similar issue when trying to rename a saved search, but this fix does not solve that problem.

**Comment by richardsd** [comment]
create_rename_saved_search

**Comment by richardsd** [comment]
Running the pipeline at https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/574/

File: elements/search/nuxeo-search-form.html [hunk 1]
```diff
@@ -797,11 +803,12 @@ <h2>[[i18n('searchForm.delete.heading')]]</h2>
             if (this._renaming) {
               this.$.renameDialog.close();
               this.$['saved-searches'].get().then(function() {
-                this.set('searches.' + (this.selectedSearchIdx - 1) + '.title', _el.data.title);
+                var searchNewIdx = this.searches.map(function(search) { return search.id; })
+                  .indexOf(this.selectedSearch.id);
```
[the remark this answers is not in the document]
Comment (richardsd) [reply elements/search/nuxeo-search-form.html:807]: Yes, much better 👍 Thanks.

**Review by mnixo — DISMISSED** [review]

**Review by Gabez0r — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**